src/app/plugins/platform/JavaScript/index.ts

```diff
@@ -75,6 +75,7 @@ class JavaScript extends Generator {
         'html-webpack-plugin',
         'clean-webpack-plugin',
         'terser-webpack-plugin',
+        'uglifyjs-webpack-plugin',
```

Comment:
Ah, okay. I added it because I was getting a build error with it was missing, but from your comment below, sounds like the fix instead is to remove/change the references to UglifyWebpackPlugin.